pages/opam-using.html

```diff
@@ -94,15 +88,16 @@ <h3>Installing CoqIDE</h3>
 <p>You may also want to install CoqIDE. Note that this requires GTK+
 development files (<code>gtksourceview3</code>) to be available on the
 system. As for Coq, names of system packages to install can be
-determined using <code>opam-depext</code>:</p>
+determined using when running <code>opam install</code>:</p>
```

Comment:
```suggestion
determined when running <code>opam install</code>:</p>
```

Again, I'm wondering if it's not enough to say that opam will make sure that the required system packages are available (I can't test any of this because this feature of opam isn't supported on NixOS).

Reply:
It might be better to remove mention of this entirely. We have more dedicated documentation for installation right? This page should probably be what a first-time-user sees, and if needed we can link the correct documentation.

Reply:
I don't think we have more detailed documentation specific to Coq, but opam does have a good bunch of documentation, and indeed, suggesting to use Zulip for troubleshooting seems like a good idea.